README.rst

```diff
@@ -14,7 +14,11 @@ Contains different interactive visualisations to understand better.
 
 How to Use:
 -----------
-Switch to a virtual environment! ``cd`` into project's directory.
+Switch to a virtual environment! ``cd`` into project's directory:
+```
+virtualenv --python=<path_to_python2> venv
```

Comment:
Wasn't sure if this project is intended to be Python 3 or 2, pls comment if change is needed

Reply:
Hi! Its a completely Python3 project!